src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx

```diff
@@ -0,0 +1,43 @@
+// (C) 2020 GoodData Corporation
+import * as React from "react";
+import { shallow } from "enzyme";
+import { IConfigurationPanelContentProps } from "../ConfigurationPanelContent";
+import { DEFAULT_LOCALE } from "../../../../constants/localization";
+import { VisualizationTypes } from "../../../../constants/visualizationTypes";
+import GeoPushpinConfigurationPanel from "../GeoPushpinConfigurationPanel";
+import UnsupportedProperties from "../../configurationControls/UnsupportedProperties";
+
+describe("GeoPushpinConfigurationPanel", () => {
+    function createComponent(props: IConfigurationPanelContentProps) {
```

Comment:
please add the return type for this function. 
It should be `ShallowWrapper`

Reply:
updated